File: app/src/main/java/com/zillennium/utswap/screens/setting/profileQRCodeScreen/ProfileQRCodeActivity.java
package com.zillennium.utswap.screens.setting.profileQRCodeScreen;

import android.os.Bundle;
import android.widget.ImageView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.zillennium.utswap.bases.BasePassedActivity;
import com.zillennium.utswap.databinding.ActivitySettingProfileQrcodeBinding;


public class ProfileQRCodeActivity extends AppCompatActivity {

    ActivitySettingProfileQrcodeBinding binding;

    @Nullable
    protected void onCreate(Bundle savedInstanceState){
        super.onCreate(savedInstanceState);

        binding = ActivitySettingProfileQrcodeBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());


        // Set Passed Back
        ImageView ivBack = binding.ivBack;
        BasePassedActivity.onPassedBack(ProfileQRCodeActivity.this, ivBack);
    }

}

File: app/src/main/java/com/zillennium/utswap/screens/setting/profileQRCodeScreen/ProfileQRCodeActivity.kt
package com.zillennium.utswap.screens.setting.profileQRCodeScreen

import com.zillennium.utswap.R
import com.zillennium.utswap.bases.mvp.BaseMvpActivity
import com.zillennium.utswap.databinding.ActivitySettingFundPasswordBinding
import com.zillennium.utswap.databinding.ActivitySettingProfileQrcodeBinding
import java.io.IOException

class ProfileQRCodeActivity :
    BaseMvpActivity<ProfileQRCodeView.View, ProfileQRCodeView.Presenter, ActivitySettingProfileQrcodeBinding>(),
    ProfileQRCodeView.View {

    override var mPresenter: ProfileQRCodeView.Presenter = ProfileQRCodePresenter()
    override val layoutResource: Int = R.layout.activity_setting_profile_qrcode

    override fun initView() {
        super.initView()
        try {
            binding.apply {

                // Set Passed Back
                ivBack.setOnClickListener {
                    onBackPressed()
                }

            }
            // Code
        } catch (error: IOException) {
            // Must be safe
        }
    }
}